### PR TITLE
feat(Anchor): create styles for Sbanken

### PR DESCRIPTION
**Pull request by joakbjerk**
## Summary

Creating a theme file for sbanken anchor styling.

## Fix at a later date

- Add icons according to figma sketches
- Create a proper contrast styling in line with sbanken brand

### CODE REVIEW

File: packages/dnb-eufemia/src/elements/anchor/__tests__/themes/Anchor.sbanken.screenshot.test.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  })
+
+  // "active" simulation is suddenly too unstable – no reason found
+  // it('have to match the anchor-contrast "active" state', async () => {
+  //   const screenshot = await makeScreenshot({
+  //     selector: '[data-visual-test="anchor-contrast"]',
+  //     simulate: 'active',
+  //   })
+  //   expect(screenshot).toMatchImageSnapshot()
+  // })
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/elements/anchor/__tests__/themes/Anchor.sbanken.screenshot.test.ts:117]: I think we can try to enable this test, also for UI. We can do this also in a follow up PR.

File: packages/dnb-eufemia/src/elements/anchor/style/themes/dnb-anchor-theme-sbanken.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+@use '../anchor-mixins.scss';
+
+.dnb-anchor {
+  font-weight: var(--font-weight-medium);
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/elements/anchor/style/themes/dnb-anchor-theme-sbanken.scss:4]: Use rather the -sb- version here.

**Review by tujoworker — APPROVED** [review]
Looks good 👍 just some small changes needed as mentioned in the comments.

File: packages/dnb-eufemia/src/elements/anchor/__tests__/themes/Anchor.sbanken.screenshot.test.ts (hunk 1 above)
Comment (joakbjerk) [reply packages/dnb-eufemia/src/elements/anchor/__tests__/themes/Anchor.sbanken.screenshot.test.ts:117]: Okay, lets do it in a follow-up PR 👍

**Pull request merged**